viewers/nerf_viewer/README.md

```diff
@@ -0,0 +1,85 @@
+# Nerf Viewer Quick Start
+
+## 1 Client
+
+### 1.1 Install Dependencies
+
+#### 1.1.1 Install NVM
+
+Please follow the instructions in:
+
+https://juejin.cn/post/7120267871950733320/
```

Comment:
Do we have any installation instruction in English? Say https://github.com/nvm-sh/nvm#installing-and-updating.

Reply:
The original version of README.md is for debug purpose. In the lastest doc, the [installation instructions](https://github.com/WYK96/xrprimer/blob/5b76711b953035e6e1fc3191efe19f66a4fbecf3/docs/en/web_renderer/xrnerf/README.md?plain=1#L27) has been updated so that it is suitable for non-chinese users.